pkg/kubelet/kubelet_pods_test.go

```diff
@@ -6196,7 +6196,7 @@ func TestKubelet_HandlePodCleanups(t *testing.T) {
 			podWorkers, _, processed := createPodWorkers()
 			kl.podWorkers = podWorkers
 			originalPodSyncer := podWorkers.podSyncer
-			syncFuncs := newPodSyncerFuncs(originalPodSyncer)
+			syncFuncs := newTestPodSyncerFuncs(originalPodSyncer)
```

Comment:
hm why is this in this PR? sorry if I'm missing something but I don't see this referenced elsewhere

Reply:
I just saw it on the way and it threw me off when I was searching for something. It's not important, but I felt like fixing it.

Reply:
if you happen to do another push, can you drop this commit and add it to a separate PR? it is already a large PR so the fewer things we can fit for the alpha impl the better IMO

Reply:
done, I also unexported the fsPullRecordsAccessor

Reply:
I'll create a separate commit later, I think there might've been other things that could just live in tests